docs/index.md

```diff
@@ -7,7 +7,7 @@ Get what you need, not more, not less:
 | Command                        |  Dependencies |
 |:-------------------------------|---------------|
 | `pip install zen3geo`          | rioxarray, torchdata |
-| `pip install zen3geo[raster]`  | rioxarray, torchdata, xbatcher |
+| `pip install zen3geo[raster]`  | rioxarray, torchdata, xbatcher, xpystac, zarr |
 | `pip install zen3geo[spatial]` | rioxarray, torchdata, datashader, spatialpandas |
 | `pip install zen3geo[stac]`    | rioxarray, torchdata, pystac, pystac-client, stackstac |
```

Comment:
Debating on whether to put `xpystac` under the 'raster' or 'stac' extras :thinking:

Reply:
Decided to go with `xpystac` under the 'stac' extras for now (done at commit 49c29b1d0fb4cc45b60084198208f883700c9648) since `xpystac` depends on `pystac`, `xarray`, and somewhat optionally on `stackstac`, so the Venn diagram overlaps more with the STAC ecosystem than raster-like stuff like xarray and Zarr.